src/QualificationRunner.Core/QualificationRunnerConfiguration.cs

```diff
@@ -18,15 +18,20 @@ public class QualificationRunnerConfiguration : OSPSuiteConfiguration, IQualific
    {
       public override string ProductName => Constants.PRODUCT_NAME_WITH_TRADEMARK;
 
+      public QualificationRunnerConfiguration() : base(Assembly.GetExecutingAssembly())
+      {
+      }
+
+
       //not used in this context
       public override int InternalVersion { get; } = 1;
       public override Origin Product { get; } = Origins.Other;
       public override string ProductNameWithTrademark => Constants.PRODUCT_NAME_WITH_TRADEMARK;
-      public override string IconName { get; } = ApplicationIcons.Comparison.IconName;

```

Comment:
Why did this change to a hard coded string?

Reply:
no idea